packages/pods-core/src/builtin/GDocPod.ts

```diff
@@ -0,0 +1,273 @@
+import { ImportPod, ImportPodConfig, ImportPodPlantOpts } from "../basev3";
+import { JSONSchemaType } from "ajv";
+import { PodUtils } from "../utils";
+import axios from "axios";
+import { googleDocsToMarkdown } from "docs-markdown";
+import _ from "lodash";
+import {
+  DendronError,
+  DVault,
+  NoteProps,
+  NoteUtils,
+  stringifyError,
+  DEngineClient,
+} from "@dendronhq/common-all";
+
+const ID = "dendron.gdoc";
+
+type GDocImportPodCustomOpts = {
+  /**
+   * google docs personal access token
+   */
+  token: string;
+
+   /**
+   * document Id of doc to import
+   */
+  documentId: string;
+
+  /**
+   * name of hierarchy to import into
+   */
+  fname: string;
+
+   /**
+   * import comments from the doc in text or json format
```

Comment:
Please add a note on the default behavior when this is not set (and also to corresponding the dendron-site docs)

Reply:
Fname is the **required** field to specify the hierarchy for note

Reply:
Fname is the **required** field to specify the hierarchy for note